### ISSUE TITLE
Android: Cannot delete custom local calendar

### ISSUE THREAD

**Issue by creambyemute** [EddyVerbruggen/nativescript-calendar#6]
Steps to reproduce:
1. Start app on a device
2. Create calendar event in a new local custom calendar (by providing calendar: {name: "someName"})
3. Go to the system calendar app/your preferred calendar app
4. Try to delete the event (you can't)
5. Try to delete the calendar (you can't)

The only way I've found to delete the events in the custom/local calendar as well as the custom/local calendar is to go to the settings --> apps --> show system processes --> calendar storage --> delete cache

Haven't tried it on iOS though as I would have to setup everything first in order to deploy it to a device.

**Comment by EddyVerbruggen** [comment]
Interesting. Wondering if that's how Google intended it or if we need to set some sort of property on the created calendar so it's deletable by other apps.

**Comment by EddyVerbruggen** [comment]
The Calendar was created with default permissions which used to be fine, but I'm now explicitly setting the acces level to 'owner'. I can confirm on Android 7 that makes a difference; I used to have a read only event, now I can edit and delete these events.

Not sure how to delete __any__ Calendar from the Calendar system app, but I've added a `deleteCalendar` function to the plugin for our convenience.

All available in 1.2.0 on both iOS and Android.
